src/main/java/frc/team2412/robot/subsystems/LEDSubsystem.java

```diff
@@ -1,21 +1,39 @@
 package frc.team2412.robot.subsystems;
 
+import static frc.team2412.robot.Hardware.*;
+import static frc.team2412.robot.subsystems.LEDSubsystem.LEDConstants.*;
+
 import edu.wpi.first.wpilibj.AddressableLED;
 import edu.wpi.first.wpilibj.AddressableLEDBuffer;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class LEDSubsystem extends SubsystemBase {
-	private final int BUFFER_LENGTH = 72;
+	// CONSTANTS
+	public static class LEDConstants {
+		public static final int BUFFER_LENGTH = 72;
+
+		// private final RGB PURPLE_RGB = 145, 48, 255;
```

Comment:
The other place the colors are set.

Reply:
everything be in intake game piece type enum now